reload.js

```diff
@@ -4,20 +4,24 @@
   var loc     = location;
   var host    = loc.hostname + (loc.port ? ':' + loc.port : '');
   var timeout = setTimeout;
-  var ref, check, xhr, lastModified, time;
+  var check, xhr, lastModifiedOnServer, lastModifiedDocument;
   var waitingTime1 = 1000 * 4;      // 4 seconds
   var waitingTimeX = 1000 * 60 * 5; // 5 minutes
+
   if (loc.host === host) {
-    ref = new Date().getTime();
     check = function () {
       xhr = new XMLHttpRequest();
       xhr.onload = function () {
-        lastModified = this.getResponseHeader('Last-Modified');
-        time = new Date(lastModified).getTime();
-        if (time > ref) {
-          loc.reload(true);
-        } else {
-          timeout(check, waitingTimeX);
+        lastModifiedOnServer = this.getResponseHeader('Last-Modified');
+        lastModifiedDocument = document.lastModified;
+        if (lastModifiedOnServer && lastModifiedDocument) {
```

Comment:
Can you explain why this is needed? The document lastModified timestamp is almost guaranteed to always be different from the server last modified. Also, this feature isn't necessarily available in all browsers.

Reply:
I tested with the latest versions of the browsers Firefox, Chrome, Opera, Edge and IE on Windows 10.
All of them behaved the same way.
Date and time from the server and from the current document I wrote to the console were the same.
When I changed the files on the server, all tested browsers reloaded the pages as expected.